video/out/w32_common.c

```diff
@@ -109,6 +112,9 @@ struct vo_w32_state {
     HHOOK parent_win_hook;
     HWINEVENTHOOK parent_evt_hook;
 
+    HMENU menu;
+    void *menu_ctx; // talloc context for MENUITEMINFOW.dwItemData
```

Comment:
This is not freed anywhere

Reply:
It's created with `w32` as parent talloc context.

Reply:
Ah, indeed, I still have memory leak tho. Didn't look closer into it. Indeed it is not this one, but `mdata` copies.